src/System.Globalization/tests/CultureInfo/CultureInfoTests.netstandard1.7.cs

```diff
@@ -0,0 +1,122 @@
+            
+            ci = new CultureInfo(lcid, false);
+            Assert.Equal(ci.Name, cultureName);
+            Assert.Equal(ci.LCID, lcid);
+            Assert.Equal(ci.UseUserOverride, false);
+            Assert.Equal(ci.IsReadOnly, false);
+
+            ci = CultureInfo.GetCultureInfo(lcid);
+            Assert.Equal(ci.Name, cultureName);
+            Assert.Equal(ci.LCID, lcid);
+            Assert.Equal(ci.UseUserOverride, false);
+            Assert.Equal(ci.IsReadOnly, true);
+            
+            ci = CultureInfo.GetCultureInfo(cultureName);
+            Assert.Equal(ci.Name, cultureName);
+            Assert.Equal(ci.LCID, lcid);
+            Assert.Equal(ci.UseUserOverride, false);
+            Assert.Equal(ci.IsReadOnly, true);
+
+            ci = CultureInfo.GetCultureInfo(cultureName, "");
+            Assert.Equal(ci.Name, cultureName);
+            Assert.Equal(ci.LCID, lcid);
+            Assert.Equal(ci.UseUserOverride, false);
+            Assert.Equal(ci.IsReadOnly, true);
+            Assert.Equal(ci.TextInfo, CultureInfo.InvariantCulture.TextInfo);
+            Assert.Equal(ci.CompareInfo, CultureInfo.InvariantCulture.CompareInfo);
+
+            ci = CultureInfo.CreateSpecificCulture(cultureName);
+            Assert.Equal(ci.Name, specificCultureName);
+
+            ci = CultureInfo.GetCultureInfoByIetfLanguageTag(cultureName);            
+            Assert.Equal(ci.Name, cultureName);
+        }
+
+        [Fact]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public void InstalledUICultureTest()
+        {
+            // as we didn't change current UI culture, Installed UI culture should match current UI culture
+            Assert.Equal(CultureInfo.InstalledUICulture, CultureInfo.CurrentUICulture);
+        }
+
+        [Fact]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public void GetCulturesTest()
+        {
+            foreach (CultureInfo ci in CultureInfo.GetCultures(CultureTypes.NeutralCultures))
+                Assert.True(ci.IsNeutralCulture || ci.Equals(CultureInfo.InvariantCulture));
+
+            foreach (CultureInfo ci in CultureInfo.GetCultures(CultureTypes.SpecificCultures))
+                Assert.False(ci.IsNeutralCulture);
+        }
+
+        [Fact]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public void ClearCachedDataTest()
+        {
+            CultureInfo ci = CultureInfo.GetCultureInfo("ja-JP");
+            Assert.True((object) ci == (object) CultureInfo.GetCultureInfo("ja-JP"));
+            ci.ClearCachedData();
+            Assert.False((object) ci == (object) CultureInfo.GetCultureInfo("ja-JP"));
+        }
+
+        [Fact]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public void CultureNotFoundExceptionTest()
```

Comment:
never mind, already in existing Ctor_String_Invalid()

---
In reply to: [78286935](https://github.com/dotnet/corefx/pull/11594#discussion_r78286935) [](ancestors = 78286935)